docs/source/user_guide/automl.ipynb

```diff
@@ -336,7 +365,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.4"
+   "version": "3.7.5"
```

Comment:
I think thanks to @freddyaboulton's additions you can run `make lint-fix` locally and this will get cleaned up!!